INSTALL.rst

```diff
@@ -87,6 +87,14 @@ important: ``trunk_nic`` must match your choice of trunk NIC in the "Networking
 For a detailed description of the configuration needed for various switch
 setups, see ``docs/network-drivers.md``.
 
+Logging level and directory can be set in the ``[general]`` section.
+``log_dir`` shouldn't end with a trailing slash and the HaaS user must have
+ownership of the specified directory.::
```

Comment:
this line "user must have ownership ..." should go in the documentation about in may be docs/logging.md